README.md

```diff
@@ -1,19 +1,55 @@
 Editing Task Force
 =================
 
-These are specifications and explainers related to the work of the Editing Task Force. 
+Editing Task Force is considered to be a part of [W3C WebApps Working Group](https://w3c.github.io/webappswg/). 
 
-**Actively developed specs:**
+Its purpose is to gather and incubate emerging issues in the area of rich text editor development. 
 
-* See the [Editing Taskforce Charter](CHARTER.md)
+Details can be found in the [Group Charter](https://github.com/w3c/editing/blob/gh-pages/CHARTER.md).
 
-* See the [Selection API](https://w3c.github.io/selection-api/) specification, which defines the `Selection` interface, the `getSelection()` method, and the `selectstart` and `selectionchange` events.
+## Actively developed specs
 
+* [Content Editable Disabled](https://github.com/w3c/editing/blob/gh-pages/Active%20Documents/contentEditableDisabled.html)
+* [Virtual Keyboard Policy ](https://github.com/MicrosoftEdge/MSEdgeExplainers/blob/master/VirtualKeyboardPolicy/explainer.md)
+* [Selection API](https://w3c.github.io/selection-api/)
 
-**These specs are incomplete and it is not expected that they will advance beyond draft status:**
+## Graduated
+[Highlight API](https://drafts.csswg.org/css-highlight-api-1/)
```

Comment:
Probably want a little note here saying what "graduated" means? Same with "backlog" below.

Reply:
will be fixed in the next commit.